### PR TITLE
Accomodate for Different Headsign Naming

### PR DESCRIPTION
**Pull request by AdrianaCeric**
Not all route pattern names follow the `'49 to <Destination> (<destid>)[ from <Origin> (<originid)]'` format. This PR adds an additional conditional that checks if the route name doesn't have an open bracket and uses the route's `shortName` to generate the correct headsign name.

Co-authored by: Miles G.

### CODE REVIEW

File: lib/util/viewer.js [hunk 1]
```diff
@@ -83,6 +84,17 @@ export function extractHeadsignFromPattern(pattern) {
     }
   }
 
+  // In some cases the string generated by OTP doesn't include a " "
+  // see https://github.com/opentripplanner/OpenTripPlanner/blob/7ce72cf554e6cf469d683869db5684eae3ce86f8/src/main/java/org/opentripplanner/graph_builder/module/TripPatternNamer.java#L72-L183
+  // In this case, we need to rely on the routeShortName to determine what to remove
+  if (isBlank(headsign) && routeShortName) {
+    console.log(routeShortName)
```
Comment (miles-grant-ibigroup) [inline lib/util/viewer.js:91]: Whoops I think this slipped in!

Comment (AdrianaCeric) [reply lib/util/viewer.js:91]: Lol. good catch

**Review by binh-dam-ibigroup — CHANGES_REQUESTED** [review]
Could you add a unit test for this change? (so we can see relevant examples in the code).

**Comment by AdrianaCeric** [comment]
@binh-dam-ibigroup I wrote unit tests, where should I put them?

**Comment by binh-dam-ibigroup** [comment]
> @binh-dam-ibigroup I wrote unit tests, where should I put them?

@AdrianaCeric Put them in new file `__tests__/util/viewer.js`. (See other js files in `__tests__` for how unit tests are written.)

**Review by binh-dam-ibigroup — APPROVED** [review]
Nice, thanks for the changes!

**Review by philip-cline — APPROVED** [review]
LGTM. Unit tests were helpful and well written. Happy to see sesame street get some rapid transit.

**Pull request merged**